bittensor/_neuron/text/core_server/__init__.py

```diff
@@ -92,7 +92,8 @@ def __init__(
         causallmnext = None,
         seq2seq = None,
         synapse_list = None,
-        netuid = None
+        netuid = None,
+        blacklist_hotkeys = [],
```

Comment:
Should use `None` here rather than a mutable arg: https://stackoverflow.com/questions/1132941

Reply:
Nice catch! Fixed